src/SFA.DAS.EmployerFinance.UnitTests/MessageHandlers/EmployerAccounts/AddedPayeSchemeEventHandlerTests.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Messages.Events;
+using SFA.DAS.EmployerFinance.Application.Commands.AddAccountPayeScheme;
+using SFA.DAS.EmployerFinance.MessageHandlers.EventHandlers.EmployerAccounts;
+using SFA.DAS.Testing;
+
+namespace SFA.DAS.EmployerFinance.UnitTests.MessageHandlers.EmployerAccounts
+{
+    [TestFixture]
+    [Parallelizable]
+    public class AddedPayeSchemeEventHandlerTests : FluentTest<AddedPayeSchemeEventHandlerTestsFixture>
+    {
+        [Test]
+        public Task Handle_WhenHandlingCreatedAccountEventAddedPayeSchemeEvent_ThenShouldSendAddAccountCommand()
```

Comment:
It also shouldn't contain CreatedAccountEvent 😉! Fixed.

I don't have particularly strong views on the naming of tests, I just try to keep them consistent across the solution. I like the idea of making the names more generic, so that they don't contain the names of symbols - as you say, that's not refactor friendly (although Rider/Resharper will search comments and strings during a rename refactor, so I'm surprised it doesn't offer this).

It might be a bit hard to remove all symbols from the test names though and still make them meaningful!